app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -72,20 +73,24 @@ def profile_or_identity_needs_verification?
     end
 
     def track_authorize_analytics(result)
-      analytics_attributes = result.to_h.except(:redirect_uri).
-                             merge(user_fully_authenticated: user_fully_authenticated?)
+      analytics_attributes =
+        result.to_h.except(:redirect_uri).merge(user_fully_authenticated: user_fully_authenticated?)
 
-      analytics.track_event(
-        Analytics::OPENID_CONNECT_REQUEST_AUTHORIZATION, analytics_attributes
-      )
+      analytics.track_event(Analytics::OPENID_CONNECT_REQUEST_AUTHORIZATION, analytics_attributes)
     end
 
     def identity_needs_verification?
-      ((@authorize_form.ial2_requested? || @authorize_form.ial2_strict_requested?) &&
-        (current_user.decorate.identity_not_verified? ||
-        decorated_session.requested_more_recent_verification?)) ||
-        (@authorize_form.ial2_strict_requested? &&
-        !current_user.active_profile&.includes_liveness_check?)
+      (
+        (@authorize_form.ial2_requested? || @authorize_form.ial2_strict_requested?) &&
+          (
+            current_user.decorate.identity_not_verified? ||
+              decorated_session.requested_more_recent_verification?
+          )
+      ) ||
+        (
+          @authorize_form.ial2_strict_requested? &&
+            !current_user.active_profile&.includes_liveness_check?
+        )
```

Comment:
Certain things like this are a good example of where we'd still have control in the formatting, and where it might make sense to rewrite the code anyways by splitting out new variables or methods to simplify the condition. At which point, the subsequent formatting would likely be more sensible.